star-war-character-app/src/pages/Vehicle/VehicleDetails/VehicleDetails.tsx

```diff
@@ -0,0 +1,101 @@
+import { Link, useParams } from "react-router-dom";
+import { IRootState, useAppDispatch } from "../../../redux/store";
+import { useSelector } from "react-redux";
+import { Fragment, useEffect } from "react";
+import { getIndividualVehicleActions } from "../../../redux/VehicleSlice/VehicleAsyncThunk";
+import { Strings } from "../../../resource/Strings";
+import { Loader } from "../../../Loader/Loader";
+import moment from "moment";
+import "./VehicleDetails.css"
+
+
+const VehicleDetails = () => {
+  const dispatch = useAppDispatch();
+  const { vehicleId } = useParams();
+  const isLoading = useSelector(
+    (state: IRootState) => state.vehicleStateData.isLoading
+  );
+  const { specificVehicle } = useSelector(
+    (state: IRootState) => state.vehicleStateData
+  );
+  useEffect(() => {
+    const id = Number(vehicleId);
+    dispatch(getIndividualVehicleActions({ id }));
+  }, [dispatch, vehicleId]);
+  return (
+    <div>
+    <Fragment>
+      {isLoading ? (
+        <Loader />
+      ) : (
+        <>
+          <div className="vehicleDetailsContainer">
+            <div className="detail-item">
+              <label>{Strings.name}:</label> {specificVehicle.name}
+            </div>
+
+            <div className="detail-item">
+              <label>{Strings.model}:</label> {specificVehicle.model}
+            </div>
+
+            <div className="detail-item">
+              <label>{Strings.vehicleClass}:</label> {specificVehicle.vehicle_class}
+            </div>
+
+            <div className="detail-item">
+              <label>{Strings.consumables}:</label>{" "}
+              {specificVehicle.consumables}
+            </div>
+
+            <div className="detail-item">
+              <label>{Strings.cargoCapacity}:</label>{" "}
+              {specificVehicle.cargo_capacity}
+            </div>
+
+            <div className="detail-item">
+              <label>{Strings.costInCredits}:</label>{" "}
+              {specificVehicle.cost_in_credits}
+            </div>
+
+            <div className="detail-item">
+              <label>{Strings.manufacturer}:</label>
+              {specificVehicle.manufacturer}
+            </div>
+
+            <div className="detail-item">
+              <label>{Strings.length}:</label> {specificVehicle.length}
+            </div>
+
+            <div className="detail-item">
+              <label>{Strings.crew}:</label> {specificVehicle.crew}
+            </div>
+
+            <div className="detail-item">
+              <label>{Strings.passengers}:</label> {specificVehicle.passengers}
+            </div>
+
+            <div className="detail-item">
+              <label>{Strings.maxAtmosphereSpeed}:</label>
+              {specificVehicle.max_atmosphering_speed}
+            </div>
+
+            <div className="detail-item">
+              <label>{Strings.created}:</label>{" "}
```

Comment:
These should be corrected, and should not be part of the code

Reply:
I have resolved this issue in this PR https://github.com/Harsh-Upadhyay-AT/Star-War-Character-App/pull/2 Please check it.